dist/common/scripts/scylla_util.py

```diff
@@ -351,6 +351,183 @@ def user_data(self):
         except urllib.error.HTTPError:  # empty user-data
             return ""
 
+class azure_instance:
+    """Describe several aspects of the current Azure instance"""
+
+    EPHEMERAL = "ephemeral"
+    ROOT = "root"
+    GETTING_STARTED_URL = "http://www.scylladb.com/doc/getting-started-azure/"
+    META_DATA_BASE_URL = "http://169.254.169.254/1.0/meta-data/instance/"
+    ENDPOINT_SNITCH = "GossipingPropertyFileSnitch"
```

Comment:
Do we have the property file? in https://github.com/scylladb/scylla/pull/8361/files#diff-9fe4dbe99d9ee03a478c9b25175e06bbd4cef9303c0ab1b9bdeea6e52d284cd4R141 we use simpleSnitch which doesn't require one

Reply:
this is a very good comment
so we will need to add Azure Snitch to Scylla too!

Reply:
I don't think we shall use SimpleSnitch, it's not production ready snitch
so until Azure Snitch is in Scylla whoever uses this snitch later in image, will need to create the property file

Reply:
@tarzanek There is a template file already in Scylla repo - `conf/cassandra-rackdc.properties` - so we can actually edit this file with the required information